samples/msal-node-samples/AGC-README.md

```diff
@@ -0,0 +1,22 @@
+The non-AGC E2E tests are not able to be run in the AGCE. In order to run - and ONLY run - the AGC E2E tests while in the AGCE, changes must be made to the jest-preset files. Inside of jest-preset.js and jest-preset-no-setup.js, line 7 must be commented and line 9 must be uncommented.
```

Comment:
I would suggest, instead of commenting/uncommenting the presets file, use the command line flags to override the preset in AGC. You can add a script in the `package.json` e.g. `npm run test:agc` to make this easier

Reply:
great idea, thanks